packages/rest/src/sequence.ts

```diff
@@ -116,3 +116,77 @@ export class DefaultSequence implements SequenceHandler {
     }
   }
 }
+
+/**
+ * A sequence implementation using middleware chains
+ */
+export class MiddlewareSequence implements SequenceHandler {
+  /**
+   * Constructor: Injects findRoute, invokeMethod & logError
+   * methods as promises.
+   *
+   * @param invokeMiddleware - invoker for registered middleware in a chain.
+   * To be injected via SequenceActions.INVOKE_MIDDLEWARE.
+   */
+  constructor(
+    @inject(SequenceActions.INVOKE_MIDDLEWARE)
+    readonly invokeMiddleware: InvokeMiddleware,
+    @config()
+    readonly options: InvokeMiddlewareOptions[] = [],
+  ) {
+    if (this.options.length === 0) {
+      this.options = [
+        {
+          chain: 'middlewareChain.rest',
+          orderedGroups: [
```

Comment:
`LogErrorProvider ` is not included here

Reply:
See https://github.com/strongloop/loopback-next/pull/5366#discussion_r422257512